common/core/desktop/tests/unit/kmx/031 - caps lock.kmn

```diff
@@ -24,3 +24,11 @@ group(Main) using keys
 
 + 'b' > 'pass.'
 + 'B' > 'fail.'
+
+c Caps is on. The virtual key should match.
++ [SHIFT K_C] > 'fail.'
++ [K_C] > 'pass.'
+
+c Caps was pressed and released, so Caps is now off.  The virtual key should still match.
++ [SHIFT K_D] > 'fail.'
++ [K_D] > 'pass.'
```

Comment:
It's not clear to me exactly what you are testing here. Are you purposely ignoring the caps state for the virtual keys? (If so, should update the comments to clarify.)

Otherwise:

```suggestion
c Caps is on. The virtual key should match.
+ [SHIFT K_C] > 'fail.' c either ncaps or caps
+ [NCAPS K_C] > 'fail.'
+ [CAPS K_C] > 'pass.'

c Caps was pressed and released, so Caps is now off.  The virtual key should still match.
+ [SHIFT K_D] > 'fail.' c either ncaps or caps
+ [CAPS K_D] > 'fail.'
+ [NCAPS K_D] > 'pass.'
```

Reply:
I reverted this file.

Reply:
I reverted this file.